### PR TITLE
DTS-38827 project setup bredcrum anf flow change

### PR DESCRIPTION
**Pull request by brahmanand1**

### CODE REVIEW

**Review by prabasak23 — APPROVED** [review]

File: UI/src/app/config/project-config/basic-config/basic-config.component.css [hunk 1]
```diff
@@ -89,4 +90,28 @@ width: 40%;
 
 .form-container .p-invalid {
   color: #B00020;
+}
+
+:host ::ng-deep input{
+  height: 41px;
```
Comment (HinPublicis) [inline UI/src/app/config/project-config/basic-config/basic-config.component.css:96]: @brahmanand1 can we not give height and manage it with padding if possible

File: UI/src/app/config/project-config/basic-config/basic-config.component.css [hunk 2]
```diff
@@ -89,4 +90,28 @@ width: 40%;
...
+}
+
+button {
+  height: 34px;
```
Comment (HinPublicis) [inline UI/src/app/config/project-config/basic-config/basic-config.component.css:100]: @brahmanand1 can we not give height and manage it with padding if possible?

File: UI/src/app/config/project-config/basic-config/basic-config.component.css [hunk 3]
```diff
@@ -89,4 +90,28 @@ width: 40%;
...
+}
+
+.nav-container{
+padding-top: .8rem;
+padding-bottom: .8rem;
+padding-left: .3rem;
+border: solid 1px grey;
+font-weight: bolder;
+color: black
```
Comment (HinPublicis) [inline UI/src/app/config/project-config/basic-config/basic-config.component.css:109]: @brahmanand1 please use color constants that we have created.

File: UI/src/app/config/project-config/basic-config/basic-config.component.css [hunk 4]
```diff
@@ -89,4 +90,28 @@ width: 40%;
...
+}
+
+:host ::ng-deep .p-steps .p-steps-item.p-highlight .p-steps-number{
+  background-color: #9e9d9e;
```
Comment (HinPublicis) [inline UI/src/app/config/project-config/basic-config/basic-config.component.css:113]: @brahmanand1 please use color constants for this

File: UI/src/app/config/project-config/basic-config/basic-config.component.css [hunk 5]
```diff
@@ -89,4 +90,28 @@ width: 40%;
...
+}
+.blue-hover-link:hover {
+  color: blue;
```
Comment (HinPublicis) [inline UI/src/app/config/project-config/basic-config/basic-config.component.css:116]: @brahmanand1 please use color constants for this

**Review by HinPublicis — APPROVED** [review]

**Pull request merged**